docs/source/kubernetes/controllermode.rst

```diff
@@ -0,0 +1,395 @@
+  2022-09-14 20:33:50,559: custodian.k8s.policy:INFO Matching event against:{'operations': ['CREATE'], 'resources': 'pods', 'group': 'core', 'apiVersions': 'v1', 'scope': 'Namespaced'}
+  2022-09-14 20:33:50,560: custodian.k8s.policy:INFO Event Matched
+  2022-09-14 20:33:50,639: custodian.k8s.policy:INFO Filtered from 1 to 1 resource(s)
+  2022-09-14 20:33:50,639: custodian.k8s.policy:INFO allowing admission because on-match:warn, matched:1
+  10.0.201.111 - - [14/Sep/2022 20:33:50] "POST /?timeout=10s HTTP/1.1" 200 -
+  2022-09-14 20:33:50,640: c7n_kube.server:INFO {"apiVersion": "admission.k8s.io/v1", "kind": "AdmissionReview", "response": {"allowed": true, "warnings": ["example-warn-policy:This is a sample policy"], "uid": "2ec4911a-8a9d-4c8d-8aa5-2d3709877fd0", "status": {"code": 200, "message": "OK"}}}
+
+.. kubernetes_k8s-validator-authoring:
+
+Authoring Policies
+------------------
+
+The ``k8s-validator`` mode supports both built-in resource types in Kubernetes as well as Custom objects
+defined by Custom Resource Definitions. In addition, the mode allows you to specify different behaviors
+``on-match``: ``allow``, ``deny``, and ``warn``. In addition, you can specify which operations to
```

Comment:
Instead of just providing a list in this paragraph, could we break them out into a section that describes the behavior of each?       Like what does a `warn` look like or when would you want to use `allow`?